opal/mca/threads/qthreads/threads_qthreads_module.c

```diff
@@ -0,0 +1,92 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
+/*
+ * Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+ *                         University Research and Technology
+ *                         Corporation.  All rights reserved.
+ * Copyright (c) 2004-2005 The University of Tennessee and The University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * Copyright (c) 2004-2005 High Performance Computing Center Stuttgart,
+ *                         University of Stuttgart.  All rights reserved.
+ * Copyright (c) 2004-2005 The Regents of the University of California.
+ *                         All rights reserved.
+ * Copyright (c) 2019      Triad National Security, LLC. All rights
+ *                         reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include <unistd.h>
+
+#include "opal/mca/threads/qthreads/threads_qthreads.h"
+#include "opal/constants.h"
+#include "opal/util/sys_limits.h"
+#include "opal/util/output.h"
+#include "opal/prefetch.h"
+#include "opal/mca/threads/threads.h"
+#include "opal/mca/threads/tsd.h"
+
+struct opal_tsd_key_value {
+    opal_tsd_key_t key;
+    opal_tsd_destructor_t destructor;
+};
+
+static int opal_main_thread;
+struct opal_tsd_key_value *opal_tsd_key_values = NULL;
+static int opal_tsd_key_values_count = 0;
+
+/*
+ * Constructor
+ */
+static void opal_thread_construct(opal_thread_t *t)
+{
+}
```

Comment:
@npe9 do you plan to put some code in here?  
Otherwise this will just be a placeholder for now.  full qthreads support will be part of a different PR.

Reply:
I'll fix that. Does the library link now? That was what I was blocked on.

Reply:
I don't know.  I've not tried using qthreads.

Reply:
I'll check it today. Last I check there was a bug in the configure script that caused libqthread to not be included. If we get that going I'll work out the nits in the module.